src/Bridge/Symfony/Routing/RouteNameResolver.php

```diff
@@ -40,7 +42,7 @@ public function getRouteName(string $resourceClass, bool $collection): string
             $operation = $route->getDefault(sprintf('_api_%s_operation_name', $operationType));
             $methods = $route->getMethods();
 
-            if ($resourceClass === $currentResourceClass && null !== $operation && (empty($methods) || in_array('GET', $methods))) {
+            if ($resourceClass === $currentResourceClass && null !== $operation && (empty($methods) || in_array('GET', $methods, true))) {
```

Comment:
I prefer to keep in sync with what we do in Symfony (we remove `empty`, `count` and so on).